### DISCUSSION TITLE
假如我是想使用src引入alova应该怎么做？

### DISCUSSION THREAD

**Discussion by wintsa123** [alovajs/alova#273] [General]
rt，我的项目因为无法用本地npm/yarn安装，只能用src引入

**Comment by JOU-amjs** [comment]
https://alova.js.org/zh-CN/tutorial/getting-started/first-request#%E5%9C%A8%E9%9D%99%E6%80%81-html-%E4%B8%AD%E4%BD%BF%E7%94%A8
